testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -149,6 +146,12 @@ Feature: PXE boot a Retail terminal
     And I click on "Add Systems"
     Then I should see a "1 systems were added to SERVERS server group." text
 
+  Scenario: Move the image to the branch server
```

Comment:
Why was this moved? Is this image used or needed for the retail tests? it seems to be more related to the build image feature where it was before :thinking:

Reply:
Because image-sync depends on the branch server group created in previous scenario of this feature.
(there is a compatibility fallback, so without the group it does not fail, but generate a warning message in the rhn log)